setup.py

```diff
@@ -55,7 +55,7 @@
     python_requires='>=3.6',
     include_package_data=True,
     install_requires = [
-        'jupyter_server~=1.1',
+        'jupyter_server~=1.4',
```

Comment:
I think this bump is needed for https://github.com/jupyter-server/jupyter_server/pull/415

Reply:
:+1: